### PR TITLE
External links in menu

### PR DESCRIPTION
**Pull request by dzienisz**
make Summit link open in new tab and underline it 

Screenshot 2024-07-18 at 16 49 47

Screenshot 2024-07-18 at 16 52 32

### CODE REVIEW

File: src/components/Navigation.tsx [hunk 1]
```diff
@@ -32,9 +32,11 @@ export const Navigation = () => {
 													item.current
 														? 'bg-gray-900 text-white'
 														: 'text-white hover:bg-green/80 hover:text-purple',
+													item.external ? 'underline' : '',
```
Comment (ssynowiec) [inline src/components/Navigation.tsx]: why only external link underline
image

File: src/components/Navigation.tsx [hunk 2]
```diff
@@ -32,9 +32,11 @@ export const Navigation = () => {
...
 													'rounded-md px-3 py-2 font-medium',
 												)}
 												aria-current={item.current ? 'page' : undefined}
+												target={item.external ? '_blank' : undefined}
```
Comment (ssynowiec) [inline src/components/Navigation.tsx:38]: 👍

File: src/components/Navigation.tsx (hunk 1 above)
Comment (dzienisz) [reply src/components/Navigation.tsx]: Summit is the only external link and move to other website. What do you think so it it? Or maybe add a external icon next to it?

Comment (ssynowiec) [reply src/components/Navigation.tsx]: I think external icon is better idea

**Pull request merged**